app/views/layouts/application.html.haml

```diff
@@ -19,6 +19,7 @@
       = title
 
     = stylesheet_pack_tag 'common', media: 'all'
+    = stylesheet_pack_tag (current_account&.user&.setting_theme || Setting.default_settings['theme']), media: 'all'
```

Comment:
Please be sure to add theme dependent `stylesheet_pack_tag` to embedded.html.haml as well.